### PR TITLE
Enabled tests for object dataization output from `MainTests`

### LINKED ISSUE #2979: MainTest.java:43-45: Enable all disabled test in the...
The puzzle `2931-021739d9` from #2931 has to be resolved: 

https://github.com/objectionary/eo/blob/97054f31cc87f264dcc140718d5cd410835d4924/eo-runtime/src/test/java/org/eolang/MainTest.java#L43-L45

The puzzle was created by rultor on 18-Mar-24. 

Estimate: 30 minutes,  role: DEV. 

If you have any technical questions, don't ask me, submit new tickets instead. The task will be \"done\" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

### PR DESCRIPTION
**Pull request by c71n93**
Closes #2979 

---

## PR-Codex overview
This PR enables previously disabled tests in `MainTest` class by refactoring them to work without the removed `tupled-stdout.eo` object. 

### Detailed summary
- Refactored `deliversCleanOutput` test to work with `true` object dataization
- Refactored `executesJvmFullRun` test to work with `false` object dataization

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by c71n93** [comment]
@maxonfjvipon could you check this one, please?

**Review by maxonfjvipon — APPROVED** [review]

**Comment by c71n93** [comment]
@yegor256 could you check this one, please?

**Comment by yegor256** [comment]
@rultor merge

**Comment by yegor256** [comment]
@rultor merge

**Comment by c71n93** [comment]
@yegor256, @maxonfjvipon looks like something wrong with `jtcop`. Daily workflow failed for a similar reason https://github.com/objectionary/eo/actions/runs/8994999514/job/24709290008.

**Comment by maxonfjvipon** [comment]
@c71n93 just restart your CI and you'll see

**Comment by c71n93** [comment]
@maxonfjvipon I alredy merged up to date master locally and build failed with the same error.

**Comment by maxonfjvipon** [comment]
@c71n93 just do it please) or wait until #3175 is merged, I fixed it there

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**
